1d-burgers/inf_cont_burgers.py

```diff
@@ -28,16 +28,20 @@
     # Data size on the solution u
     hp["N_u"] = 100
     # Collocation points size, where we’ll check for f = 0
-    hp["N_f"] = 10000
+    # [mtsokol] for bigger values slows down drastically
+    hp["N_f"] = 100
     # DeepNN topology (2-sized input [x t], 8 hidden layer of 20-width, 1-sized output [u]
-    hp["layers"] = [2, 20, 20, 20, 20, 20, 20, 20, 20, 1]
+    # [mtsokol] 7x7=49 for one_hot_encoding, more about it in neuralnetwork.py
+    # [mtsokol] IMPORTANT! when modifying input size remember to modify BSplineLayer(num_elem=) param. (Should be done in one place only. Will fix it)
+    # [mtsokol] num_elem = sqrt(layers[0]) - 2 (only for one_hot_encoding)
+    hp["layers"] = [49, 20, 20, 20, 20, 20, 20, 20, 20, 1]
     # Setting up the TF SGD-based optimizer (set tf_epochs=0 to cancel it)
     hp["tf_epochs"] = 100
     hp["tf_lr"] = 0.03
     hp["tf_b1"] = 0.9
     hp["tf_eps"] = None
     # Setting up the quasi-newton LBGFS optimizer (set nt_epochs=0 to cancel it)
-    hp["nt_epochs"] = 200
+    hp["nt_epochs"] = 100
```

Comment:
TBH I never really was impressed with LBGFS, Adam does the job well more often than not (esp for experimentation)

Reply:
Sure, here it's more handy to test with lbgfs turned off. Should I commit this?

Reply:
It kinda over complicates the code as so yeah I would say so.